data-raw/healthy-lives/children-and-young-people/primary-pupil-absences.R

```diff
@@ -0,0 +1,47 @@
+# ---- Load packages ----
+library(tidyverse)
+library(httr)
+library(readODS)
+library(dplyr)
+library(readxl)
+
+# ---- Scrape primary schools URL ----
+#Source: https://www.gov.wales/absenteeism-primary-schools-september-2022-august-2023
+download.file("https://www.gov.wales/sites/default/files/statistics-and-research/2024-03/absenteeism-from-primary-schools-september-2022-to-august-2023-113.ods", 
+              destfile = "absenteeism.ods")
```

Comment:
You are saving the file in the package folder, please could you save as a temp folder? I'm also unable to run the rest of the script as it tells me the file is corrupted. Could you amend the situation?